Client/src/Components/Profile/ProfileView.tsx

```diff
@@ -191,6 +197,8 @@ export function ProfileView() {
                 <TableBody >
                   <ProfileGraphStateList
                     graphDataset={savedGraphState}
+                    handleRemoveGraphSet={handleRemoveGraphSet}
+                    userID={user.orcID}
```

Comment:
I dont think this should be user.orcID. We dont use this for any sort of Identifiication in the backend

Reply:
there shouldnt be user Id at all

Reply:
i added the orcID because of the the way the graphController shows that it takes a graphID and userID
![image](https://user-images.githubusercontent.com/25268105/108661331-88ddfd00-7499-11eb-9813-ef0b07dc6889.png)